### ISSUE TITLE
Making it self-learn?

### ISSUE THREAD

**Issue by pokeball99** [magahet/peon#1]
yea it would probably be cool to do that.

**Comment by magahet** [comment]
What would it learn? I've thought about implementing Q learning with reward states based on amassing a balance of useful items (enchanted tools/armor, food, beacons, etc.). That would take quite a bit of work to model the states and rewards though.

**Comment by pokeball99** [comment]
yes i though of make making it a reward system,though not sure if that would work.plus what would we want it to do with overflow items,throw them on the ground,or just be a hoarder and make chest for every thing.both would cause lag.
